src/Ring-Core/RGEnvironmentAnnouncer.class.st

```diff
@@ -116,7 +116,7 @@ RGEnvironmentAnnouncer >> methodAdded: aMethod [
 RGEnvironmentAnnouncer >> methodRemoved: aMethod [ 
 
 	"TODO:"
```

Comment:
Can re remove the TODO comment?
It does not say what is there to TODO, so it's pretty useless :)

Reply:
Also maybe your change is what was there to TODO? :)

Reply:
Thanks!